app/Http/Controllers/Admin/ProductController.php

```diff
@@ -3,30 +3,26 @@
 namespace App\Http\Controllers\Admin;
 
 use App\Http\Controllers\Controller;
-use App\Http\Requests\Product\ProductRequest;
-use App\Http\Services\Product\ProductAdminService;
-use App\Models\Product;
-use Illuminate\Http\Request;
+use App\Http\Requests\ProductRequest;
+use App\Services\Product\ProductAdminService;
 use Illuminate\Support\Facades\Session;
 
+
 class ProductController extends Controller
 {
-    protected $productServices;
-
-
-    //Display a listing of the resource.
+    private $productAdminService;
```

Comment:
why this name include 'Admin'?

Reply:
I thought this controller is used to add and update products in the Admin so I named it like that, but it doesn't seem right. I will make corrections